lib/markdown-renderer.ts

```diff
@@ -17,6 +17,16 @@ class MyOptions implements HtmlToTextOptions {
                 .replace(/./g, "=");
             return `${heading}\n${underline}\n\n`;
         },
+        blockquote(elem: any, fn: any, options: any): string {
+            const indentOptions = Object.assign({}, options);
+            // decrease word wrap, but only to a minimum of 20 columns/line
+            indentOptions.wordwrap = Math.max(20, options.wordwrap - 2);
+
+            const block: string = fn(elem.children, indentOptions);
+            return block.replace(/^>/mg, ">>") // add to quote
+                .replace(/^(?!>|$)/mg, "> ")   // new quote
+                .replace(/(^|\n)(\n)/g, "$1>$2"); // quote empty lines
```

Comment:
If we add `(?!$)` to this regex, do we get rid of the ugly trailing quotes?

Reply:
Yes it does.  Done!